tests/temp_adhoc_terms_testt.R

```diff
@@ -22,7 +22,7 @@
 #   set.vertex.attribute(net,"var_2",as.character(apply(rmultinom(100,1,rep(1/2,2)),2,FUN = function(x){which(x==1)})))
```

Comment:
Is this file needed anymore or have these test scenarios been moved to the tests folder?

Reply:
will remove, I will be adding these tests in the testthat style back in as I add more stats